x-pack/test/saved_object_api_integration/common/fixtures/es_archiver/saved_objects/spaces/data.json

```diff
@@ -195,7 +195,7 @@
         "description": "",
         "version": 1,
         "kibanaSavedObjectMeta": {
-          "searchSourceJSON": "{\"index\":\"91200a00-9efd-11e7-acb3-3dab96693fab\",\"filter\":[],\"query\":{\"query\":\"\",\"language\":\"lucene\"}}"
```

Comment:
Just for clarification, this change is required because the old archive had Visualizations referencing index patterns which didn't actually exist, and previously this wasn't an issue because we never tried to retrieve them?